### ISSUE TITLE
Confusion about Baseline

### ISSUE THREAD

**Issue by JaycgbEDC** [tedhuang96/nirrt_star#5]
Hi!
    I'm very grateful for the code you have provided.As you mentioned in your paper, you considered NRRT*-GNG as one of the baselines.
    When I read the paper of NRRT*, I found they used image and other attributes like stepwise as input.And in the part of model, they used CNN model and other ways like atrous convolution and ASPP and so on.
   While reading your code(train_unet.py), I found your model completely different with NRRT*.And I want to know whether you have modified by yourself to enhance the efficiency?
    Thank you!

**Comment by tedhuang96** [comment]
Hi!

Sorry for the delay. Thanks for this great question! Our research motivation for this work is mainly on the innovation of integration between sampling-based planning algorithms and learning-based guidance. In our work, we do not specifically emphasize the contributions on the design of the neural networks for the boost of overall planning performance. This is the main reason why we chose a vanilla ResNet based U-Net for Neural RRT* and a vanilla PointNet++ for Neural Informed RRT*.

Regarding the extra features of step-size as input in the original Neural RRT* paper, this is a contribution which the authors of Neural RRT* claim for their works, which is that they can design a unified neural network to adapt to different configuration parameters of the planning problem. As they already show that it works, we do not have to make effort to claim the same point in our work. Thus, we did not include test on different step sizes, nor include these components in either the baseline design or the pointnet++ design. 

I indeed agree with you on that having a different design of the neural network could change the planning performance to some extent. I definitely encourage you to try to exactly replicate the design of Neural RRT* and report the results with our datasets as benchmarks. My guess is that the performance at the first few iterations could be improved if you finetune with the hyperparameters. However, in terms of the slightly longer iterations, Neural Informed RRT* will outperform Neural RRT* as long as the capacity of the neural networks deployed by two algorithms are not wildly different from each other, and this is due to our innovation on the algorithm integration for improving optimal convergence rate.

Feel free to follow up with any additional questions and happy to discuss! Again sorry for the delayed response, and I will try my best to be responsive.
